tools/sdk-tools/adpcm/vadpcm_enc.c

```diff
@@ -132,7 +132,7 @@ int main(int argc, char **argv)
 
     inBuffer = malloc(16 * sizeof(s16));
 
-    fread(&FormChunk, sizeof(Chunk), 1, ifile);
+    if (fread(&FormChunk, sizeof(Chunk), 1, ifile)){};
```

Comment:
Same as above. Use `(void) fread(...)` in this file too, or actually check the return value. Although I am still fucking adamant that the warning should be disabled because it's pretty irrelevant here.

Reply:
I'd rather actually *fix*the warnings rather than disable them tbh. It's not hard at all to do

Reply:
Properly *fixing* it would mean actually checking for the return value of `fscanf` and `fread` and aborting if it's incorrect. The `(void)` thing and your thing too are exactly the same as just silencing it, because they don't fix the underlying problem.
Now this "problem" isn't much of a problem at all in the first place, because you don't really expect this to eat random user supplied files, and even if it does, it will likely fail in a different place anyway, so in my opinion it is fine to leave the ignore flag in there.

Reply:
the (void) doesn't actually silence the warning in my case, but the way this does it does. In either case, I agree that it'd need to abort here in the case that the return value is incorrect

Reply:
Again resolved by using these as error-checkers.